test/jdk/java/io/ByteArrayInputStream/ReadAllReadNTransferTo.java

```diff
@@ -48,8 +48,16 @@ public static void main(String... args) throws IOException {
         int position = random.nextInt(SIZE/2);
         int size = random.nextInt(SIZE - position);
 
-        ByteArrayInputStream bais =
-            new ByteArrayInputStream(buf, position, size);
+        ByteArrayInputStream bais = new ByteArrayInputStream(buf);
+        bais.readAllBytes();
+        if (bais.read(new byte[0]) != -1) {
+            throw new RuntimeException("read(byte[]) did not return 0");
+        }
+        if (bais.read(new byte[1], 0, 0) != -1) {
+            throw new RuntimeException("read(byte[],int,int) did not return 0");
```

Comment:
Should these exception messages be "did not return -1"?

Reply:
Oh you are correct. Thanks!